data/data/agent/systemd/units/apply-host-config.service

```diff
@@ -18,7 +18,7 @@ ExecStart=podman run --net host --cidfile=%t/%n.ctr-id --cgroups=no-conmon --log
 ExecStop=/usr/bin/podman stop --ignore --cidfile=%t/%n.ctr-id
 ExecStopPost=/usr/bin/podman rm -f --ignore --cidfile=%t/%n.ctr-id
 
-KillMode=none
+KillMode=mixed
 Type=oneshot
 RemainAfterExit=true
 
```

Comment:
Its not quite clear why we need the KillMode=mixed here for 
```
Type=oneshot
RemainAfterExit=true
```
while the KillMode was removed for `start-cluster-installation.service`

Reply:
As noted in the [commit message](https://github.com/openshift/installer/pull/9562/commits/c19ffdd455b56cef8a1e82da2dc68cb63ac983d2), it's consistent with what quadlet does. Here's what they have to say:

https://github.com/containers/podman/blob/d1d8f3334fea2394a17958ab9f04429d865789e4/pkg/systemd/quadlet/quadlet.go#L579-L580

Basically this gives conmon another chance to shut down the pod before systemd `kill -9`s everything.